valkyrie/spec/valkyrie/storage_adapter_spec.rb

```diff
@@ -26,4 +26,26 @@
       expect(storage_adapter).to have_received(:find_by).with(id: "yo")
     end
   end
+
+  describe ".adapter_for" do
+    it "finds a storage adapter for a given identifier" do
+      file = instance_double(Valkyrie::StorageAdapter::StreamFile, id: "yo")
+      allow(storage_adapter).to receive(:handles?).and_return(true)
+      described_class.register(storage_adapter, :find_test)
+
+      expect(described_class.adapter_for(id: file.id)).to eq storage_adapter
+    end
+  end
+
+  describe ".delete_id" do
```

Comment:
Now I wonder if it should just be called `delete`, since `id` is a named parameter; also would be symmetric to the method in the metadata persisters. Or `delete_by` to be symmetric to `find_by`?